es6-arrow-functions/main.js

```diff
@@ -0,0 +1,49 @@
+const $jokeContainer = document.querySelector('#joke-container');
+const $flashImage = document.querySelector('#flash-image');
+const $jokeForm = document.querySelector('#joke-form');
+
+const jokester = {
+  tellJoke: function (setup, punchline) {
+    $jokeForm.classList.add('d-none');
+    const $introStatement = renderJokePhrase('Hey Flash...');
+    appendJokePhrase($introStatement);
+    setTimeout(function () {
+      const $jokeSetup = renderJokePhrase(setup);
+      appendJokePhrase($jokeSetup);
+      setTimeout(function () {
+        const $jokePunchline = renderJokePhrase(punchline);
+        appendJokePhrase($jokePunchline);
+        flash();
+      }, 2000);
+    }, 2000);
+  }
+};
+
+const renderJokePhrase = phrase => {
+  const $phrase = document.createElement('h4');
+  $phrase.textContent = phrase;
+  $phrase.className = 'fade-in text-center mb-3 col-12';
+  return $phrase;
+};
+
+const appendJokePhrase = $phrase => {
+  $jokeContainer.append($phrase);
+};
+
+const flash = () => {
+  const laughingUrl = 'images/flash-laugh.gif';
+  laugh(laughingUrl);
+};
+
+const laugh = url => {
+  setTimeout(function () {
+    $flashImage.setAttribute('src', url);
+  }, 3000);
+};
```

Comment:
We don't want to completely refactor the code, the idea is to leave it the way it was but change some of the `function`s to arrow functions to make it work as is.